bin/dump.sh

```diff
@@ -157,6 +160,28 @@ combine_pkcs12() {
   fi
 }
 
+convert_keys_to_rsa() {
+  if [[ "${CONVERT_KEYS_TO_RSA}" != yes ]]; then
+    return
+  fi
+
+  if [[ -z "${DOMAIN}" || "${#DOMAINS[@]}" -gt 1 ]]; then
+    local outputdir_subdirs=(${outputdir}/*/)
+    for subdir in "${outputdir_subdirs[@]}"; do
+      local i=$(basename "${subdir}" /)
+      if [[ -f ${outputdir}/${i}/cert.pem && -f ${outputdir}/${i}/key.pem ]]; then
+        log "Converting key for domain ${i} to RSA key file"
+        openssl rsa -in ${outputdir}/"${i}"/key.pem -out ${outputdir}/"${i}"/rsakey.pem
```

Comment:
```suggestion
        openssl rsa -in "${outputdir}/${i}/${privatekey_file}" -out "${outputdir}/${i}/rsakey.pem"
```

See also previous comment. For consistency, maybe we should also introduce an environment variable for settings file name + extension of `rsakey.pem`? See https://github.com/kereis/traefik-certs-dumper/commit/cea5907733267e935a8d0b10145d672589fc30d1#diff-ed933e4c63bca6ca162f9af2745864cfea768a052ed1dd83c00f04b3b4b878e3R8 for reference.

Reply:
I am happy to take care of your suggestions concerning a new environment variable for `rsakey.pem`. I'll hope I will find the time during the next days.